frontend/src/components/Marks/styles.js

```diff
@@ -1,3 +1,4 @@
+import { Typography } from "antd";
 import styled from "styled-components";
```

Comment:
Are these stylings imported anywhere? Just realised `Marks.jsx` does not import these hahaha. Could we remove this file if this is not being imported anywhere?